src/pages/50x.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import Footer from './shared/footer/index';
+import Nav from './shared/header/nav/index';
+import CookieWarning from './shared/cookie-warning';
+
+
+const NotFoundPage = () => (
```

Comment:
I'd rename this const to something more Server error like

```suggestion
const NotFoundPage = () => (
const ServerErrorPage = () => (
```